adapter-excel/src/main/java/ch/prevo/open/node/adapter/excel/ExcelMatchNotificationListener.java

```diff
@@ -18,30 +18,23 @@
  ******************************************************************************/
 package ch.prevo.open.node.adapter.excel;
 
-import ch.prevo.open.data.api.FullCommencementNotification;
-import ch.prevo.open.data.api.FullTerminationNotification;
+import ch.prevo.open.data.api.FullMatchForCommencementNotification;
+import ch.prevo.open.data.api.FullMatchForTerminationNotification;
 import ch.prevo.open.node.data.provider.MatchNotificationListener;
 import ch.prevo.open.node.data.provider.error.NotificationException;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.time.LocalDateTime;
 
 public class ExcelMatchNotificationListener implements MatchNotificationListener {
 
     private static final Logger LOG = LoggerFactory.getLogger(ExcelMatchNotificationListener.class);
 
-    private static final String FILE_PROPERTY = "node.adapter.excel.out.file";
-    private static final String FALLBACK_FILE = "retirement-fund-out-data";
-    private static final String FILE_NAME_FORMAT = "%1$s_%2$tY-%2$tm-%2$td_%2$tH-%2$tM-%2$tS.%2$tL.xlsx";
 
     @Override
-    public void handleTerminationMatch(FullTerminationNotification notification) throws NotificationException {
-        final String filename = getFilename();
-
-        try (final TerminationNotificationWriter writer = new TerminationNotificationWriter(filename)) {
+    public void handleMatchForCommencementNotification(FullMatchForCommencementNotification notification) throws NotificationException {
+        try (final MatchForCommencementNotificationWriter writer = new MatchForCommencementNotificationWriter()) {
```

Comment:
Should the AbstractNotificationWriter implement AutoCloseable for this to work even better? (Vice versa for handleMatchForTerminationNotification)

Reply:
I am not sure I understand this comment. AbstractNotificationWriter implements Closeable, which extends AutoCloseable. What would be the advantage of implementing AutoCloseable instead?

Reply:
Ow! Thanks for that info ;-)